misago/markup/tests/test_parser.py

```diff
@@ -192,7 +192,7 @@ def test_clean_current_link(self):
 """.strip()
 
         expected_result = """
-<p>Lorem ipsum: <a href="/" rel="nofollow">test.com</a></p>
```

Comment:
It makes sense to keep internal links without nofollows. I'll consult this with fellow seo magicans and see.